packages/components/app/styles/components/dropdown.scss

```diff
@@ -44,13 +44,7 @@
 
   &:focus,
   &.is-focus {
```

Comment:
@MelSumner you have to remove the parent declaration/selector (otherwise there are two "focus" in the generated CSS selector)

instead of this:
```
  &:focus,
  &.is-focus {
    @include hds-focus-ring-with-pseudo-element($top: -1px, $right: -1px, $bottom: -1px, $left: -1px, $radius: 5px);
  }
```
should be this:
```
  @include hds-focus-ring-with-pseudo-element($top: -1px, $right: -1px, $bottom: -1px, $left: -1px, $radius: 5px);
```